Benqi-Agents/Large-Delegate-Votes-Change/README.md

```diff
@@ -0,0 +1,37 @@
+# Delegate Votes Monitor Agent
+
+## Description
+
+This agent detects large (>= 30%) increments in a delegate account's QI token
+vote balance by monitoring `DelegateVotesChanged` events emitted by the token
+contract.
+
+## Supported Chains
+
+- Avalanche
+
+## Alerts
+
+- BENQI-1-1
```

Comment:
Since we have only one finding, there's no need to have the sub-number in the alert Id, `BENQI-1` is enough imo